README.md

```diff
@@ -50,14 +50,14 @@ Invoke-WebRequest -Uri "https://github.com/microsoft/sbom-tool/releases/latest/d
 
 ##### Linux (curl)
 
-```bash
+```shell
 curl -Lo sbom-tool https://github.com/microsoft/sbom-tool/releases/latest/download/sbom-tool-linux-x64
 chmod +x sbom-tool
 ```
 
 ##### macOS (curl)
 
-```bash
+```shell
```

Comment:
Visually identical, and bash is appropriate for any macos environment (see https://github.com/highlightjs/highlight.js/blob/main/SUPPORTED_LANGUAGES.md)